darkside-test-lib/src/androidTest/java/cash/z/ecc/android/sdk/darkside/test/DarksideTestCoordinator.kt

```diff
@@ -301,27 +297,44 @@ class DarksideTestCoordinator(val wallet: TestWallet = newDarksideTestWallet())
             "https://raw.githubusercontent.com/zcash-hackworks/darksidewalletd-test-data/master/basic-reorg/after-small-reorg.txt"
         private const val largeReorg =
             "https://raw.githubusercontent.com/zcash-hackworks/darksidewalletd-test-data/master/basic-reorg/after-large-reorg.txt"
-        private const val DEFAULT_START_HEIGHT = 663150
+
         private const val DEFAULT_SEED_PHRASE =
             "still champion voice habit trend flight survey between bitter process artefact blind carbon truly provide dizzy crush flush breeze blouse charge solid fish spread"
 
         /**
          * @return A wallet appropriate for passing into [DarksideTestCoordinator].
          */
         fun newDarksideTestWallet(
-            alias: String = "DarksideTestCoordinator",
-            seedPhrase: String = DEFAULT_SEED_PHRASE,
-            startHeight: Int = DEFAULT_START_HEIGHT,
-            host: String = COMPUTER_LOCALHOST,
-            network: ZcashNetwork = ZcashNetwork.Mainnet,
-            port: Int = network.defaultPort,
+                alias: String = "DarksideTestCoordinator",
+                seedPhrase: String = DEFAULT_SEED_PHRASE,
+                startHeight: Int = DarksideNetwork.DEFAULT_START_HEIGHT,
```

Comment:
In the future, will we expect test cases to pass in a start height that's different from the default?  I'm basically asking whether we need this method parameter or not.

Reply:
We actually should, because the correct usage of this tool involves being able to use a different set of activation heights. We haven't due to priorities/time constraints but the support for this parameter seems adecquate.